applications/serial_lte_modem/doc/slm_testing.rst

```diff
@@ -61,15 +61,20 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
       :class: highlight
 
       **AT#XSLEEP=?**
-      #XSLEEP: (0,1)
+      #XSLEEP: (0,1,2)
       OK
 
+      **AT#XSLEEP**
+
+   You can quit the Idle status by GPIO.
+
       **AT#XSLEEP=1**
 
-   Reset the kit to exit sleep mode.
-   If you are testing with an external MCU and :option:`CONFIG_SLM_GPIO_WAKEUP` is enabled, you can wake up by GPIO as well.
+   If you are testing with :option:`CONFIG_SLM_GPIO_WAKEUP` enabled, you can wake up by GPIO.
 
+      **AT#XSLEEP=2**
 
+   You can power on UART by GPIO.
```

Comment:
Remove these (covered in the suggestion above).

Reply:
Updated